broker/CMakeLists.txt

```
@@ -0,0 +1,26 @@
+project(beerocks_broker VERSION ${prplmesh_VERSION})
+message("${BoldWhite}Preparing ${BoldGreen}${PROJECT_NAME}${BoldWhite} for the ${BoldGreen}${TARGET_PLATFORM}${BoldWhite} platform${ColourReset}")
+
+# Global definitions
+add_definitions(-DBEEROCKS_VERSION="${prplmesh_VERSION}")
+add_definitions(-DBEEROCKS_BUILD_DATE="${BUILD_DATE}")
+add_definitions(-DBEEROCKS_REVISION="${PRPLMESH_REVISION}")
+add_definitions(-DBEEROCKS_BIN_PATH="${INSTALL_PATH}/bin/")
+add_definitions(-DBEEROCKS_CONF_PATH="${INSTALL_PATH}/config/")
```

I realize this is just copied from other CMakeLists, but could we do the right thing here? I.e.
```suggestion
add_definitions(-DBEEROCKS_BIN_PATH="${CMAKE_INSTALL_FULL_BINDIR}/")
add_definitions(-DBEEROCKS_CONF_PATH="${CMAKE_INSTALL_PREFIX}/config/")
```
For the conf path we should actually use SYSCONFDIR, but since the rest of the config is not stored there, let's keep it like this.

---

Done